.github/workflows/testing.yml

```diff
@@ -0,0 +1,70 @@
+name: testing
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - '*'
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool:
+          - '1.10'
+          - '2.8'
+          - '2.9'
+
+    steps:
+      - name: Clone the connector
+        uses: actions/checkout@v2
+
+      - name: Setup tarantool ${{ matrix.tarantool }}
+        uses: tarantool/setup-tarantool@v1
+        with:
+          tarantool-version: ${{ matrix.tarantool }}
+
+      - name: Setup golang for the connector and tests
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.13
+
+      - name: Run base tests
+        run: |
+          mkdir snap xlog
```

Comment:
Сleaning up thеsе folders after test seems like a good approach. But since there are not likely be two different tarantools per same folder for tests due to `go test .` approach, this is optional.

Reply:
Yep, each test suite (base, queue, uuid, multi) runs in its own suite folder, so there is no need to clean up these folders because they are always new for the test suite.